seismiqb/crop_batch.py

```diff
@@ -1463,3 +1464,68 @@ def show_roll(self, n=1, components=None, zoom_slice=None, displayed_name=None,
             **kwargs
         }
         return plot_image(data, **kwargs)
+
+    def show_frequencies(self, indices=(0, ), src='images', trace_indices=((0, 0), (-1, -1)), axis=2, d=None,
+                         displayed_name=None, **kwargs):
+        """ Show Fourier frequency spectrum of a component. X-axis of the plot corresponds to frequency
+        values in Hz while y-axis stands for amplitudes of specific frequencies.
+
+        For the analysis the method selects specific traces and item-indices. Traces are 1d slices taken along an axis.
+
+        Uses units of Nyquist frequency when depicting the spectrum. In this way achieves consistency with
+        action `bandpass_filter`.
+
+        Parameters
+        ----------
+        indices: int or tuple or list
```

Comment:
```suggestion
        indices: int or sequence of int
```
Is more consistent with other docstrings.